src/test/resources/com/puppycrawl/tools/checkstyle/grammar/javadoc/InputDosLineEndingAsNewlineMultiline.javadoc

```diff
@@ -0,0 +1,7 @@
+* line 0
+* Checks that commentContent starts with '*' javadoc comment identifier.
+    *&lt;property name="ignorePattern" value="^ *\* *[^ ]+$"/&gt*
+* This * is not a leading *
+* @return true if commentContent starts with '*' javadoc comment identifier
+* line 5
+    * line 6
```

Comment:
Needed this for test coverage.

Reply:
@rnveach I need to add this one to git attributes, though, thanks for bringing this up.